app/models/api/v3/converter_stats_presenter.rb

```diff
@@ -18,9 +18,10 @@ class ConverterStatsPresenter
 
       attr_reader :key
 
-      def initialize(key, gql)
-        @key = key
-        @gql = gql
+      def initialize(key, gql, graph_attributes)
+        @key              = key
+        @gql              = gql
+        @graph_attributes = graph_attributes || ATTRIBUTES
```

Comment:
I don't see much point in `|| ATTRIBUTES` any more; the request should supply a list of all attributes wanted. If something like...

```json
{
    "keys": {
        "transport_car_using_electricity": []
    }
}
```

... is requested, just return no attributes; apparently the sender of the request doesn't want anything!

We could actually fail such a request with `400 Bad Request` or `422 Unprocessable Entity`, since it's probably an mistake to ask for no attributes.